spec/models/cookbook_spec.rb

```diff
@@ -433,7 +433,9 @@ def generate_params(opts = {})
               'yum' => '~> 2.1.3'
             },
             source_url: opts[:source_url],
-            issues_url: opts[:issues_url]
+            issues_url: opts[:issues_url],
+            chef_version: [['12.4.1', '12.4.2'], ['11.2.3', '12.4.3']],
+            ohai_version: [['8.8.1', '8.8.2'], ['8.9.1', '8.9.2']]
```

Comment:
okay, so yeah everything needs to be pluralized, including the wire format here.